tests/network/ping.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: iputils
+# Summary: Ping another machine
```

Comment:
This leads one to believe that this is a new test for ping, and we already have one of those, one can argue that the tests have different purposes, and in the end I see no point in adding one more MM scenario, just for the sake of it.

Reply:
This test is required to be cloned in a nested test to check our MM worker setup scripts in the openQA test distribution.
It also makes sense to run it on tw to ensure that it works outside of the nested openQA-in-openQA environment.

Reply:
> we already have one of those

At least I didn't find a MM ping test.

Reply:
That this is a ping test? I thought I did.
Anyway this file is not part of the PR anymore as I followed @foursixnine 's suggestion to move it into the setup_multimachine module.